src/lib.rs

```diff
@@ -1708,44 +1719,28 @@ fn encode_frame(sequence: &mut Sequence, fi: &mut FrameInvariants, fs: &mut Fram
             None => (),
         }
     } else {
-        let obu_extension = 0;
+        let tile = encode_tile(fi, fs); // actually tile group
+        let obu_payload_size = tile.len() as u64;
         let mut buf1 = Vec::new();
         {
             let mut bw1 = BitWriter::<BE>::new(&mut buf1);
-            bw1.write_obu_header(OBU_Type::OBU_FRAME, obu_extension).unwrap();
+            // uleb128()
+            let mut coded_payload_length = [0 as u8; 8];
```

Comment:
I originally used 16 bytes here, like = [0 as u8; 16];
but 8 would be still fine, for 8 bytes, it is 7 x 8 = 56 bits, so 2 ^ 56 size in bytes.

Reply:
In the libaom spec it's actually not allowed to be more than 2^32-1 bytes.

Reply:
Oh, right... uleb128() is really so.
The tile_size_minus_1 uses "le(TileSizeBytes)", which has no limit in # of bytes